extension/chrome/elements/compose-modules/formatters/general-mail-formatter.ts

```diff
@@ -80,4 +89,19 @@ export class GeneralMailFormatter {
     // throws ComposerResetBtnTrigger when user closes pass phrase dialog without entering
     return await view.storageModule.decryptSenderKey(parsedSenderPrv);
   };
+
+  private static isSenderInKeyUserIds = async (view: ComposeView, signingKey?: ParsedKeyInfo | undefined): Promise<boolean> => {
```

Comment:
Probably we can make `signingKey` non-optional, as there is no reason to check sender in non-existent key.

Also in line 97 there is a check `senderEmail !== view.acctEmail && signingKey?.keyInfo.emails`, and if `signingKey` is `undefined` then we `return true` which is incorrect, as key is undefined and can't include sender

Reply:
Thanks, I made the necessary updates here.